core/modules/installer/installer.pages.inc

```diff
@@ -28,6 +28,9 @@ function installer_browser_page($type) {
       backdrop_set_message(t('You have recently installed modules. <a href="@url">Continue to the enable page.</a>', array('@url' => url('admin/installer/install/enable'))), 'status', FALSE);
     }
   }
+
+  backdrop_set_title(t('Install '.$type));
```

Comment:
This won't get picked up by the translation extractor because it won't know what `$type` is. It would be better (in this case), to use either a `switch` or `if/elseif/else` statements here with each translated version.